src/views/shared/transaction-info/transaction-info.view.tsx

```diff
@@ -191,9 +201,11 @@ function TransactionInfo({
           from={myHermezAddress}
           to={{
             subtitle: "My Ethereum address",
-            value: getPartiallyHiddenEthereumAddress(
-              getEthereumAddress(transaction.fromHezEthereumAddress)
-            ),
+            value: transaction.fromHezEthereumAddress
```

Comment:
Maybe abstract this?

Reply:
Now that you mention it and I look at the 4 cases, why are we calling `getEthereumAddress` on 3 of the 4 cases?

Reply:
I see, it's to remove the `hez:` prefix from the address on L1 TXs.